stl/inc/chrono

```diff
@@ -5351,6 +5418,24 @@ namespace chrono {
 
                     _Validate_specifiers(_Spec, _Val);
 
+#if defined(_MSVC_EXECUTION_CHARACTER_SET) && _MSVC_EXECUTION_CHARACTER_SET == 65001 // TRANSITION, VSO-1468747 (EDG)
+                    if constexpr (is_same_v<_CharT, char>) {
+                        if (_Specs._Localized) {
+                            wostringstream _Wstream;
+                            _Wstream.imbue(_FormatCtx.locale());
+
+                            wchar_t _Fmt_str[4];
+                            _Chrono_spec<wchar_t> _Wspec{._Modifier = _Spec._Modifier, ._Type = _Spec._Type};
+                            _Wstream << _STD put_time<wchar_t>(&_Time, _Fmt_string(_Wspec, _Fmt_str));
+
+                            _Stream << _Convert_wide_to_narrow<char_traits<char>>(
+                                __std_code_page::_Utf8, _Wstream.view(), allocator<char>{});

```

Comment:
I don't know why testing didn't catch it, but `view` is only available in C++20 and later, and this is a C++17 feature.

Reply:
chrono formatting is C++20 only. The C++17 stuff in this header is just moving some c++17 stuff from filesystem into chrono.